lib/components/src/blocks/ArgsTable/ArgValue.tsx

```diff
@@ -30,10 +30,11 @@ const Summary = styled.div<{ isExpanded?: boolean }>(({ isExpanded }) => ({
   flexWrap: 'wrap',
   alignItems: 'flex-start',
   marginBottom: '-4px',
+  minWidth: 100,
```

Comment:
This value is a bit arbitrary. Without it, a short string like `default` would wrap the `lt` to the next line. 100px min width seemed small enough to me that the column wouldn't be overly big if there were no defaults, but would keep relatively short values from breaking to a new line.

Reply:
Fair